board/batocera/fsoverlay/etc/init.d/S11share

```diff
@@ -169,16 +169,16 @@ mountNetwork() {
     # execute all commands in /boot/batocera-boot.conf which are like : sharenetwork_cmd1=my command
     if ! grep -E '^[ ]*sharenetwork_[a-z]*[0-9][ ]*=' "${SHARECONFFILE}" |
 	    sed -e s+'^[ ]*sharenetwork_\([a-z]*\)[0-9][ ]*='+'\1 '+ |
-	    while read CTYPE CMD
+	    while read -r CTYPE CMD
```

Comment:
This can improve the scripting but let @nadenislamarre decide. I think it's usefull to map driver to directories which contains whitespace and needs to be escaped. But not my buisness

Reply:
read without -r will mangle backslashes